### ISSUE TITLE
dispatcher.utter_message is not working

### ISSUE THREAD

**Issue by azizullah2017** [RasaHQ/rasa-sdk#121]
__version__ = "1.5.0"
```
**actions.py**
# changed from dispatcher.utter_message(response)
dispatcher.utter_message(text= response)

**executor.py**
def utter_message(
        self,
        image: Optional[Text] = None,
        json_message: Dict[Text, Any] = None,
        template: Optional[Text] = None,
        attachment: Optional[Text] = None,
        text: Optional[Text] = None,
        buttons: Optional[List[Dict[Text, Any]]] = None,
        *elements: Dict[Text, Any],
        **kwargs: Any,
    ) -> None:

## if remove the comment it does not work.
message = {
            "text": text
            # "buttons": buttons,
            # "elements": elements,
            # "custom": json_message,
            # "template": template,
            # "image": image,
            # "attachment": attachment,
        }
         message.update(kwargs)

        self.messages.append(message)
```

**Comment by akelad** [comment]
@wochinge since you're looking into stuff not working with forms already, can you take a look here too?

**Comment by eric-bonfadini** [comment]
I was having this issue as well.

Looks like the method `async def _utter_responses` in `rasa.core.actions.action` checks if there are keys in response (template, buttons, etc) disregarding if they are None or not.
I guess in previous versions of the sdk a message just had non-empty fields.

**Comment by wochinge** [comment]
fixed by rasa release 1.5.1 and this pr https://github.com/RasaHQ/rasa/pull/4855 🎉
